msig_contract_deployment/msig_erc20_bootstrap.md

```diff
@@ -9,6 +9,10 @@ Action 1: Update the existing evm_runtime contract, adding supports for the foll
   - *admincall action: this action allow ENF & BPs to co-sign to satisfy eosio.evm@active and act on behalf of any EVM account in emergency situations.*
   - *event mechanisim: this is the mechanisim allowing event generated in the solidity contract in EVM to be sent as bridge message via inline action in native EOS.*
 
+```
+sha256sum 66c288293d28a295aa2be75e05d92bebb25833cb3ce05b324370fb846ba3e08a  evm_runtime.wasm
+```
```

Comment:
For consistency, you should probably also add the hash of the ABI file.

Reply:
Also, mention the version of CDT used to build these so people know how to get the same hashes.

Since we cannot say v4.0.0 exactly, we could say v4.0.x and provide a link to the exact commit hash of CDT used to build.